coretex/cli/modules/scripts.py

```diff
@@ -73,16 +86,22 @@ def generateUpdateScript(config: Dict[str, Any]) -> str:
     )
 
 
-def dumpScript(updateScriptPath: Path, config: Dict[str, Any]) -> None:
+def dumpScripts(startScriptPath: Path, updateScriptPath: Path, config: Dict[str, Any]) -> None:
```

Comment:
Replace `startScriptPath` and `updateScriptPath` with a parameter of type `list`